app.js

```diff
@@ -1,17 +1,37 @@
 const express = require("express");
 const app = express();
-const { getTopics, getEndpoints } = require("./controllers/topics-controllers");
+const {
+  getTopics,
+  getEndpoints,
+  getArticleById,
+} = require("./controllers/topics-controllers");
 
-app.get("/api", getEndpoints)
+app.get("/api", getEndpoints);
 
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticleById);
+
```

Comment:
The app.all('*') is the end of the Express chain and would be better suited to be here before the error handling middleware chain